### PR TITLE
build: scraper working fine

### PR DESCRIPTION
**Pull request by heytulsiprasad**
## NB
The `scraper.js` is exported to `app.js` which runs when the server starts. *No* scripts has been changed, so now the new site will be available on `localhost:3000` after `npm run start`. 

### Any suggestions appreciated. 👍

### CODE REVIEW

**Comment by heytulsiprasad** [comment]
:camera_flash: **update imgur link for one team member** [10dc592](https://github.com/zairza-cetb/zairza-website/pull/74/commits/10dc5926ea4bf1b9d9a4739f75c7269b02dc70b3)

the link of a member has been expired which resulted in appearing of blank space, which is still visible on `zairza.in` which is now updated to an imgur direct link in this commit.

File: scraper.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+
+        // scrapContent has all blog objects inside an array
+        // TODO: Add cover image urls into cover.json
+
+        // MAKE A CONSTRUCTION FUNCTION TO STORE COVER URLS
+        const Cover = function (cover) {
+            this.cover = cover
+        }
+
+        // FIRST COVER:
```
Comment (ankitjena) [inline scraper.js:104]: Why are you fetching images separately for all the posts?

Comment (heytulsiprasad) [reply scraper.js:104]: Hey, while fetching them using a loop statement they were pushed to the array out of order which made different covers to land on different posts and it was random every time. Is there any better way out? 
**PS**: I also don't support hard-coding fetches separately.

Comment (ankitjena) [reply scraper.js:104]: Yes, read from the json array you make first. Then after you fetch the image, append it within the json object itself.

**Comment by heytulsiprasad** [comment]
Have worked on a new branch. Will update in a new PR.

**Pull request closed without merging**